docker-app/qfieldcloud/core/models.py

```diff
@@ -1035,6 +1039,17 @@ class Meta:
         validators=[MinValueValidator(1), MaxValueValidator(100)],
     )
 
+    # Packaging offliner to be used by the QGIS worker container.
+    packaging_offliner = models.CharField(
+        _("Packaging Offliner"),
+        help_text=_(
+            'The packaging offliner is used withing jobs to convert QGIS projects into QField projects. The new "Optimized Offliner" should be selected over the legacy "QGIS Core Offline Editing" for new projects.'
+        ),
+        max_length=100,
+        default=PackagingOffliner.PYTHONMINI,
```

Comment:
NOTE All new projects will use the new packaging offliner by default.

Reply:
That's a bold move, shouldn't we test it on a few chosen projects in the beginning?

Reply:
Ok, will do in phase 2.